components/main_menu/main_menu.test.jsx

```diff
@@ -3,10 +3,21 @@
 
 import React from 'react';
 import {shallow} from 'enzyme';
+import {IntlProvider} from 'react-intl';
 
 import {Constants} from 'utils/constants.jsx';
 
-import MainMenu from './main_menu.jsx';
+import {MainMenu} from './main_menu.jsx';
+
+function shallowWithIntl(Component, props) {
```

Comment:
Using the new approach with the context, you can directly use the `shallowWithIntl` function from the `tests/helpers/intl-test-helper.jsx` module.